keps/NNNN-kep-template/README.md

```diff
@@ -332,6 +343,194 @@ enhancement:
   CRI or CNI may require updating that component before the kubelet.
 -->
 
+## Production Readiness Review Questionnaire
+
+<!--
+
+Production readiness reviews are intended to ensure that features merging into
+Kubernetes are observable, scalable and supportable, can be safely operated in
+production environments, and can be disabled or rolled back in the event they
+cause increased failures in production. See more in the PRR KEP at
+https://git.k8s.io/enhancements/keps/sig-architecture/20190731-production-readiness-review-process.md
+
+The KEP must have a approver from the PRR review team.
```

Comment:
provide a link so they find the channel and reach out on slack if they have questions.